### DISCUSSION TITLE
Next Release planned?

### DISCUSSION THREAD

**Discussion by shresthaujjwal** [spring-projects-experimental/spring-boot-migrator#909] [General]
Hi just checking when is the next release planned?

**Comment by fabapp2** [comment]
Hi @shresthaujjwal 
There's no fixed date but I will try to provide a release from main soon.

**Reply by shresthaujjwal** [reply]
Thanks @fabapp2 will this include sbm openrewrite support module ?

**Reply by fabapp2** [reply]
Nope, incorporating the `sbm-support-rewrite` and with it OpenRewrite 8.5.x and Spring Boot 3.1.x is an ongoing progress (and will take a while).
See [858-incorporate-new-parser-into-core](https://github.com/spring-projects-experimental/spring-boot-migrator/tree/858-incorporate-new-parser-into-core) and [version/revamp](https://github.com/spring-projects-experimental/spring-boot-migrator/tree/version/revamp).
But snapshot releases for `ord.springframework.experimental:sbm-support-rewrite:0.1.0-SNAPSHOT` are available in repo.spring.io.

**Reply by shresthaujjwal** [reply]
thanks @fabapp2 which branch would you recommend to use if i want to contribute back

**Reply by fabapp2** [reply]
Hi @shresthaujjwal 
It depends a bit on the type of contribution.
The plan is to integrate the new parser component `sbm-support-rewrite` into most modules under `components`, see #859.
So, if you plan to contribute something in one of these modules, e.g. a recipe, `main` would be the branch and the work needs to be merged into `version/revamp` once it's working.
If you plan to help with the migration towards the new parser and the latest OpenRewrite version, then `version/revamp` is the right branch.
